lib/gssapi/sanon/negoex.c

```diff
@@ -193,7 +193,7 @@ _gssspi_sanon_exchange_meta_data(OM_uint32 *minor,
     if (major != GSS_S_COMPLETE)
 	return major;
 
-    sc->flags |= rfc4757_to_sanon_flags(init_flags);
+    sc->flags |= init_flags;
 
```

Comment:
flags MUST be restricted to RFC4757 extended GSS flags per draft-howard-gss-sanon

Reply:
Ok, fix coming.